rasterio/rio/edit_info.py

```diff
@@ -7,8 +7,10 @@
 
 from . import options
 import rasterio
-import rasterio.crs
+from rasterio._crs import CRS
```

Comment:
Yeah, that's a good point. We should probably keep the import consistent across the board.